sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/mathExpressions.scala

```diff
@@ -1584,9 +1600,17 @@ abstract class RoundBase(child: Expression, scale: Expression,
         }
       case IntegerType =>
         if (_scale < 0) {
-          s"""
-          ${ev.value} = new java.math.BigDecimal(${ce.value}).
+          if (ansiEnabled) {
+            val errorContext = getContextOrNullCode(ctx)
+            val evalCode = s"""
```

Comment:
can we use the standard scala multiple syntax that are also used in other places of the spark codebase?
```
"""
  |xxx
  |xxx""".stripMaigin
```